runtime/vm/callin.cpp

```diff
@@ -883,6 +883,24 @@ sendFromMethodDescriptorString(J9VMThread *currentThread, J9UTF8 *descriptor, J9
 				currentThread->returnValue2 = (UDATA)J9VMJAVALANGINVOKEMETHODTYPE_FROMMETHODDESCRIPTORSTRINGAPPENDARG_METHOD(vm);
 			}
 			c_cInterpreter(currentThread);
+			if (VM_VMHelpers::exceptionPending(currentThread)) {
+				j9object_t currentException = currentThread->currentException;
+				J9Class *exceptionClass = J9VMJAVALANGTYPENOTPRESENTEXCEPTION(vm);
```

Comment:
Using name `exceptionClassTNPE` offers better clarity. In addition, to achieve putting a rvalue on the left when comparing for equality later, pls define it as `J9Class * const exceptionClassTNPE`.

Reply:
done - changed to `exceptionClassTNPE`.